src/apof/portal/static/portal/css/topbar.css

```diff
@@ -61,6 +73,19 @@
 }
 
 .navbar-brand a{
-    color: white;
     text-decoration: none;
+
```

Comment:
All those blank lines like this one - please explain decision behind it or remove it.